src/main/java/de/unijena/cheminf/mortar/model/util/CollectionUtil.java

```diff
@@ -109,5 +109,47 @@ public static void sortGivenFragmentListByPropertyAndSortType(List<MoleculeDataM
             return 0;
         });
     }
+    //
+    /**
+     * Calculates a suitable initial size for instantiating a new HashMap or HashSet instance based on the number of elements
+     * supposed to be stored in it and the load factor that determines the rehash threshold.
+     * Calculation: initCap = (int) aNumberOfElement * (1/aLoadFactor) + 2
+     * <br>The initial capacity multiplied with the load factor (= rehash threshold) must be higher than the number of
+     * elements.
+     *
+     * @param aNumberOfElements number of elements supposed to be stored in the new HashMap or HashSet instance
```

Comment:
I will not comment further on these self-explanatory Java docs ;)

Reply:
Good ;)